R/geom_lineribbon.R

```diff
@@ -213,7 +211,7 @@ GeomLineribbon = ggproto("GeomLineribbon", Geom,
         }
       })
 
-    line_grobs = reduce(line_grobs, c)
+    line_grobs = Reduce(c, line_grobs)
```

Comment:
Similar to above, I *think* this might work:

```suggestion
    line_grobs = unlist(line_grobs, recursive = FALSE)
```